modules/core/src/test/java/org/apache/ignite/testframework/junits/common/KeyType.java

```diff
@@ -0,0 +1,20 @@
+package org.apache.ignite.testframework.junits.common;
+
+/**
+ * Defined type of key
+ * */
+public enum KeyType {
```

Comment:
Let's move this enum inside `GridCommonAbstractTest`

Reply:
The class `GridCommonAbstractTest `is big enough and class `KeyType `is used in other packages.
What do you say?

Reply:
We should avoid names like `KeyType`. 
Each class, even internal, should be called properly according to it's semantic.